spec/ParseServerRESTController.spec.js

```diff
@@ -133,24 +134,11 @@ describe('ParseServerRESTController', () => {
     process.env.PARSE_SERVER_TEST_DB === 'postgres'
   ) {
     describe('transactions', () => {
-      beforeEach(async () => {
-        await TestUtils.destroyAllDataPermanently(true);
```

Comment:
Is it important that `destroyAllDataPermanently` is not called anymore?

Reply:
No as destroyAllDataPermanently is called automatically after each test so it’s redundant here with the rest of the beforeEach

Reply:
Good catch